framework/tlvf/AutoGenerated/include/tlvf/ieee_1905_1/tlvDeviceInformation.h

```diff
@@ -63,9 +53,37 @@ class tlvDeviceInformation : public BaseClass
         eTlvType* m_type = nullptr;
         uint16_t* m_length = nullptr;
         sMacAddr* m_mac = nullptr;
-        uint8_t* m_info_length = nullptr;
-        sInfo* m_info = nullptr;
-        size_t m_info_idx__ = 0;
+        uint8_t* m_local_interface_list_length = nullptr;
+        cLocalInterfaceInfo* m_local_interface_list = nullptr;
```

Comment:
You're storing the list in two places, as a pointer to a buffer and as a vector to `std::shared_ptr` instances of the interfaces. That seems like a bad idea; either use a C-style buffer or the `std::vector`, but not both at once.